cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,28 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    memberId: flags.string({
+      required: true,
+      description: 'Membership ID of the validator to verify',
+    }),
+
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { memberId } = this.parse(VerifyValidatorAccountCommand).flags
+
+    await this.setPreservedState({ defaultWorkingGroup: WorkingGroups.Membership })
```

Comment:
This is fine, but this will mutate the default working group preference set by the User through CLI to `Membership` every time you run the command, so after running this command you actually want to revert that change, and re-set the default working group whatever it was set before.